### PR TITLE
Misc. Changes

### PR DESCRIPTION
**Pull request by adroition**
Blocked a few  cmds and changed blockages

### CODE REVIEW

File: src/main/resources/config.yml [hunk 1]
```diff
@@ -98,20 +98,21 @@ blocked_commands:
...
   - 'n:b:/effect:Please use /potion to set effects.'
   - 's:b:/enderchest:_'
   - 'n:b:/spreadplayers:_'
   - 'n:b:/execute:_'
   - 'n:b:/blockdata:_'
   - 'n:b:/clearhistory:_'
-  - 'n:b:/save-all:_'
-  - 'n:b:/save-on:_'
-  - 'n:b:/save-off:_'
+  - 'c:b:/save-all:_'
```
Comment (rovertdude) [inline src/main/resources/config.yml]: These need to stay blocked.
They create server backups which can end the server as it overwrites the last backup.

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -98,20 +98,21 @@ blocked_commands:
   - 's:b:/time:Server-side time changing is disabled. Please use /ptime to set your own personal time.'
   - 's:b:/gamemode:Use /creative and /survival to set your gamemode.'
   - 'c:b:/ban:_'
-  - 'c:b:/pardon:_'
+  - 't:b:/pardon:_'
```
Comment (rovertdude) [inline src/main/resources/config.yml]: Make the pardon's n:b: again please.

**Comment by rovertdude** [comment]
Resolve my comments and  this will be merged.

**Comment by adroition** [comment]
Resolved.

**Pull request merged**